shared/src/main/scala/com/raquo/domtypes/generic/defs/eventProps/MiscellaneousEventProps.scala

```diff
@@ -7,7 +7,28 @@ import com.raquo.domtypes.generic.builders.EventPropBuilder
   *
   * For type param docs see [[EventPropBuilder]]
   */
-trait MiscellaneousEventProps[EP[_ <: DomEvent], DomEvent] { this: EventPropBuilder[EP, DomEvent] =>
+trait MiscellaneousEventProps[EP[_ <: DomEvent], DomEvent, DomUIEvent <: DomEvent] { this: EventPropBuilder[EP, DomEvent] =>
+
+  /**
+    * The load event fires at the end of the document loading process. At this
+    * point, all of the objects in the document are in the DOM, and all the
+    * images and sub-frames have finished loading.
+    *
+    * MDN
+    */
+  lazy val onLoad: EP[DomUIEvent] = eventProp("load")
+
+  /**
+    * Fires when the browser window is resized
+    */
+  lazy val onResize: EP[DomUIEvent] = eventProp("resize")
```

Comment:
Why do you want to move `onResize` to this trait? Is it somehow conceptually different from other events in `WindowEventProps`? Looking at [MDN](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onresize) this event seems to be window-specific by nature.

---

Looking at `onLoad` and `onScroll`, I see that they can be applied to elements as well. It would be nice to update the comments on those to not be window-specific and to link to https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload and https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onscroll

Reply:
The reason is that I want to separate event handlers that are exclusive to `Window`, `Document`, `Element` and `HTMLElement`. All three of them implement [`GlobalEventHandlers`](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers).

But here they differ:
- `Window` additionally implements [`WindowEventHandlers`](https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers) and its own [event handlers](https://developer.mozilla.org/en-US/docs/Web/API/Window#Event_handlers), which more or less corresponds to our `WindowEventProps` (these properties are not available on `Document` or `Element`).
- `Document` also adds some own [event handlers](https://developer.mozilla.org/en-US/docs/Web/API/Document#Event_handlers). We currently do not have a event properties for those.
- `Element` has (two event handler](https://developer.mozilla.org/en-US/docs/Web/API/Element#Handlers) of its own. We currently do not have a event properties for those.
- `HTMLElement` adds the [clipboard and touch events](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement#Event_handlers). The clipboard events are defined in our `ClipboardEventProps`, we do not have touch events.

Maybe, we can also make it more clear somehow, which `EventProps` traits belong to which `EventTarget`?

Reply:
I will update the comments